packages/core/src/components/button/button.tsx

```diff
@@ -27,45 +76,21 @@ export class Button {
   @Prop() buttonType: string = 'button';
 
   /**
-   * @input {boolean} If true, activates the large button size.
-   * Type: size
-   */
-  @Prop() large: boolean = false;
-
-  /**
-   * @input {boolean} If true, activates the small button size.
-   * Type: size
+   * @input {string} The button size.
+   * Possible values are: `"small"`, `"large"`, `"default"`.
```

Comment:
is `"default"` an option? I don't see it below.

Reply:
Yeah, `"default"` shouldn't be listed there. Will remove.